op-dispute-mon/mon/detector_test.go

```diff
@@ -166,31 +168,37 @@ func TestDetector_CheckAgreement_Succeeds(t *testing.T) {
 			},
 		},
 		{
-			name:      "agree_defender_wins",
-			rootClaim: mockRootClaim,
-			status:    types.GameStatusDefenderWon,
+			name:         "agree_defender_wins",
+			rootClaim:    mockRootClaim,
+			status:       types.GameStatusDefenderWon,
+			expectStatus: types.GameStatusDefenderWon,
 			expectBatch: func(batch *detectionBatch) {
 				require.Equal(t, 1, batch.agreeDefenderWins)
 			},
 		},
 		{
-			name:   "disagree_defender_wins",
-			status: types.GameStatusDefenderWon,
+			name:         "disagree_defender_wins",
+			status:       types.GameStatusDefenderWon,
+			expectStatus: types.GameStatusChallengerWon,
 			expectBatch: func(batch *detectionBatch) {
 				require.Equal(t, 1, batch.disagreeDefenderWins)
 			},
+			expectErrorLog: true,
 		},
 		{
-			name:      "agree_challenger_wins",
-			rootClaim: mockRootClaim,
-			status:    types.GameStatusChallengerWon,
+			name:         "agree_challenger_wins",
```

Comment:
Shouldn't this be `disagree_challenger_wins`?

Reply:
er, I didn't change that bit. 😆  

The `expectBatch` is expecting `agreeChallengerWins` to be the result.  We log an error because we agree with the root claim and so expect the defender to win (ie successfully defend the output root because it is valid).  So I think the names are right.